modules/fedimint-meta-client/src/api.rs

```diff
@@ -0,0 +1,73 @@
+use fedimint_core::api::{FederationApiExt as _, FederationResult, IModuleFederationApi};
+use fedimint_core::module::{ApiAuth, ApiRequestErased};
+use fedimint_core::task::{MaybeSend, MaybeSync};
+use fedimint_core::{apply, async_trait_maybe_send};
+use fedimint_meta_common::endpoint::{
+    GetConsensusRequest, GetSubmissionResponse, GetSubmissionsRequest, SubmitRequest,
+    GET_CONSENSUS_ENDPOINT, GET_CONSENSUS_REV_ENDPOINT, GET_SUBMISSIONS_ENDPOINT, SUBMIT_ENDPOINT,
+};
+use fedimint_meta_common::{MetaConsensusValue, MetaKey, MetaValue};
+
+#[apply(async_trait_maybe_send!)]
+pub trait MetaFederationApi {
+    async fn get_consensus(&self, key: MetaKey) -> FederationResult<Option<MetaConsensusValue>>;
```

Comment:
Nit: would be good to get docs on these

Reply:
I think they're documented in `endpoint.rs` which works for now but was initially unintuitive for me